src/js/script.js

```diff
@@ -20,10 +20,24 @@ import { htmlEscape } from 'escape-goat'
  */
 
 /** @type {readonly string[]} */
-export const NETWORKS = ['arweave', 'filecoin', 'walrus']
+// TODO: Add 'arweave' and 'walrus' to the NETWORKS array once we start collecting data for them
+export const NETWORKS = ['filecoin']
```

Comment:
Let's add placeholder entries in the UI for these two networks and show "coming soon" instead of the RSR value.

Reply:
Good suggestion, I've added it in https://github.com/CheckerNetwork/leaderboard/pull/16/commits/a27583af1f7643cc02b3ccfe5efbb79be8770242